test/commands/environments/delete.test.js

```diff
@@ -4,39 +4,36 @@ const {
   getEnvironmentNotFound,
   getEnvironmentValid,
   deleteEnvironment,
-  getJob,
-  getJobFailed,
+  deleteEnvironmentFailure,
 } = require('../../fixtures/api');
 const testCli = require('./../test-cli');
 const DeleteCommand = require('../../../src/commands/environments/delete');
 
 describe('environments:delete', () => {
   describe('on an existing environment', () => {
-    describe('on a completed job', () => {
+    describe('on a succcesful removal', () => {
       it('should display environment deleted', () => testCli({
         env: testEnv,
         token: 'any',
         command: () => DeleteCommand.run(['324', '--force']),
         api: [
           getEnvironmentValid(),
           deleteEnvironment(),
-          getJob(),
         ],
         std: [
           { out: 'Environment Staging successfully deleted.' },
         ],
       }));
     });
 
-    describe('on a failed job', () => {
+    describe('on a failed removal', () => {
       it('should exit with status 1', () => testCli({
         env: testEnv,
         token: 'any',
         command: () => DeleteCommand.run(['324', '--force']),
         api: [
           getEnvironmentValid(),
-          deleteEnvironment(),
-          getJobFailed(),
+          deleteEnvironmentFailure(),
         ],
         exitCode: 1,
```

Comment:
Reading at the command code I don't get how it can exit with code 1.
Does `superagent-promise` throw an error when a request response is 403, 404, 500... ?

Reply:
Just tested it and yes it seems that super agent throws an error on error:

`Note that superagent considers 4xx and 5xx responses (as well as unhandled 3xx responses) errors by default.`